packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -93,21 +94,36 @@ export function SquadPageHeader({
           </h1>
           <div
             className={classNames(
-              'mt-1 flex flex-row items-center justify-center text-text-quaternary tablet:mt-2',
+              'mt-1 flex flex-row items-center justify-center text-text-quaternary typo-subhead tablet:mt-1',
               !shouldUseListMode && 'laptopL:justify-start',
             )}
           >
             <h2
               className={classNames(
-                'text-center text-text-tertiary typo-footnote',
+                'text-center text-text-tertiary',
                 !shouldUseListMode && 'laptopL:text-left',
               )}
             >
               @{squad.handle}
             </h2>
-            <Separator />
             {createdAt && (
-              <span className="typo-caption2">Created {createdAt}</span>
+              <>
+                <Separator />
+                <span>Created {createdAt}</span>
+              </>
+            )}
+            {!!category && (
+              <>
+                <Separator />
+                <a
+                  aria-label={`View all squads in ${category.title}`}
+                  className="text-text-link"
+                  href={`/squads/discover/${category.title.toLowerCase()}`}
```

Comment:
Let's use the id here. It can be found from `squad.category.id`. You might have to update the interface for Squad if it is not yet included.

Reply:
Okay, moving to id ✅